src/components/RepeatableActionsField/RepeatableActionsField.js

```diff
@@ -21,17 +28,40 @@ export const RepeatableActionsField = memo(({
   wrapperFieldName,
   legend,
   wrapperPlaceholder,
-  children,
+  repeatableFieldAction,
+  repeatableFieldIndex,
+  hasRepeatableFields,
+  onRepeatableActionChange,
   disabled,
+  children,
 }) => {
+  const { DELETE_EXISTING } = REPEATABLE_ACTIONS;
+
+  const [isRepeatabeFieldDisabled, setIsRepeatabeFieldDisabled] = useState(false);
```

Comment:
This variable name sounds a little bit confusing to me because you don't disable repeatable field, you disable `Add` button actually, consider renaming this to e.g. `isAddButtonDisabled`

Reply:
done!